packages/core/src/App/Components/Layout/Header/wallets/account-info-wallets.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import classNames from 'classnames';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, WalletIcon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { formatMoney } from '@deriv/shared';
+import { useStore, observer } from '@deriv/stores';
+import {
+    useActiveAccount,
+    useActiveWalletAccount,
+    useLinkedWalletsAccounts,
+    useWalletAccountsList,
+} from '@deriv/hooks';
+import { AccountSwitcherWallet } from 'App/Containers/AccountSwitcherWallet';
+import { AccountsInfoLoader } from '../Components/Preloader';
+import AccountSwitcherMobile from 'App/Containers/AccountSwitcher/account-switcher-mobile';
+import AccountInfoWrapper from '../account-info-wrapper';
+import AccountInfoIcon from '../account-info-icon';
+import WalletBadge from './wallet-badge';
+
+type TAccountInfoWallets = {
+    toggleDialog: () => void;
+    is_dialog_on: boolean;
+};
+
+type TDropdownArrow = {
+    is_disabled?: boolean;
+};
+
+type TBalanceLabel = {
+    balance?: number;
+    currency?: string;
+    is_virtual?: boolean;
+    display_code?: string;
+};
+
+type TMobileInfoIcon = {
```

Comment:
@sergei-deriv same thing here


Partial<TMobileInfoIcon>
```suggestion
type TMobileInfoIcon = { 
    currency: string;
    is_virtual: boolean;
};

Partial<TmobileInfoIcon>
```